packages/buffetjs-styles/src/components/Button/index.js

```diff
@@ -63,6 +56,9 @@ const Button = styled.button`
       border: 1px solid ${colors.blue};
       color: ${colors.blue};
       padding: 0 ${sizes.button.padding.small};
+      svg {
```

Comment:
Can you update the related stories by maybe creating a new ones so we can see theses changes?